src/controllers/listingsOrganisations.ts

```diff
@@ -0,0 +1,49 @@
+import { db } from '../database/db';
+import { asyncHandler } from '../middleware';
+import { checkOrganisationOwner, checkListingOwner, cleanseData, ErrorResponse } from '../utils';
+
+/**
+ * @desc    Create an entry in the listings-organisations table
+ * @route   POST /api/listings-organisations
+ * @access  Private
+ */
+export const createListingOrganisation = asyncHandler(async (req, res) => {
+    const { listing_id, organisation_id } = req.body;
```

Comment:
For this endpoint, the permission should be set on either: `Admin` OR `Listing Owner` OR `Organisation Owner`.

You may use something like this:
```
    if (req.user.role !== 'admin' || !isListingOwner || !isOrganisationOwner) {
         return next(new ErrorResponse('Not authorised to create listing organisation as you are not the organisation or listing owner', 403));
     }
```